packages/components/src/FormField/FormField.css

```diff
@@ -216,20 +216,23 @@
 }
 
 .textarea .label {
-  top: var(--field--padding-top);
+  top: 0;
```

Comment:
@MichaelParadis given that the mini labels are in the same place with the same styling across all inputs, would this be simpler to apply to all mini labels vs just text areas?

Reply:
It's very scoped to this component and is the difference of a background colour on the label along with spacing it from the top using padding or position — don't worry, be happy!

The reason I'm bringing it up is it eliminates conditions we need to test against. Very easy to miss something like:

![image](https://user-images.githubusercontent.com/10064579/234940447-ca8a80f0-a45a-4b3f-958b-aefc51a14eed.png)

Happens on large multi-line inputs, due to mixing padding and top position.

Reply:
@fagnanm great catch. I will take a look into this!